Sofa/framework/Helper/src/sofa/helper/ComponentChange.cpp

```diff
@@ -26,31 +26,13 @@ namespace sofa::helper::lifecycle
 {
 
 const std::map<std::string, Deprecated, std::less<> > deprecatedComponents = {
-    {"MatrixMass", Deprecated("v19.06", "v19.12")},
```

Comment:
Deprecated component that are Removed should go in deprecatedComponents with the Removed() messag